core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/Entitlements.java

```diff
@@ -480,6 +503,8 @@ public static EntitlementManager load(@Nullable ManagementContext mgmt, Brooklyn
             return minimal();
         } else if ("user".equalsIgnoreCase(type)) {
             return user();
+        } else if ("powerUser".equalsIgnoreCase(type) || "power_user".equalsIgnoreCase(type)) {
+            return powerUser();
```

Comment:
Do we have constants for these to re-use?

Reply:
No for now. If we need to reuse the name of the entitlement manager we can create, but if not, I think is more clear to see directly the value here.